### PR TITLE
[RDY] fix(github-democrate): Increase github democrate lambda timeout

### PR DESCRIPTION
**Pull request by deuzu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,5 +13,5 @@ Website of 24chevres.com
   - folks in 24chevres organization will vote the PR (with :+1: and :-1: on the PR's description)
   - prefix the PR's description with `[RDY]` when the PR is ready to be merged
   - wait 24 hours after the last commit
-  - if vote result is positive, the PR will be merged
+  - if vote result is positive (2 more :+1: than :-1:), the PR will be merged
```
Comment (izeau) [inline README.md]: unclear

**Review by izeau — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (deuzu) [reply README.md]: How about that?
> the PR will be merged when the majority +1 voted positively

Comment (deuzu) [reply README.md]: it's bad :D halp!

Comment (izeau) [reply README.md]: “If there are more :+1: reactions than :-1:, the PR will be merged”?

Comment (deuzu) [reply README.md]: 2 more are needed

2 :+1: and 1 :-1:  won't merge

Comment (deuzu) [reply README.md]: Where is @msavournin when we need him?

Comment (msavournin) [reply README.md]: the pull request is validated if there is 2 (+1) more than there are (-1)

File: README.md [hunk 2]
```diff
@@ -6,12 +6,12 @@ Website of 24chevres.com
...
+## How to contribute
 
-  - fork the project
-  - do some crazy shit and propose a pull request
-  - folks in 24chevres organization will vote the PR (with :+1: and :-1: on the PR's description)
-  - prefix the PR's description with `[RDY]` when the PR is ready to be merged
-  - wait 24 hours after the last commit
-  - if vote result is positive, the PR will be merged
-  - code is then automatically deployed upon merge
+- fork the project
```
Comment (deuzu) [inline README.md:11]: markdown linter asked for this indentation 😕

**Review by izeau — APPROVED** [review]

**Pull request merged**